src/components/DisplayBalance/DisplayBalance.js

```diff
@@ -0,0 +1,104 @@
+import { ReactComponent as GrowSymbol } from '../../assets/growSymbol.svg'
+import { ReactComponent as ShareSymbol } from '../../assets/shareSymbol.svg'
+import { ReactComponent as SpendSymbol } from '../../assets/spendSymbol.svg'
+import styled from 'styled-components/macro'
+import PropTypes from 'prop-types'
+
+DisplayBalance.propTypes = {
+  type: PropTypes.string.isRequired,
+  value: PropTypes.number,
+}
+
+export default function DisplayBalance({ type, value }) {
+  return (
+    <Wrapper>
+      <CircleValueWrapper>
+        <Circle type={type} data-testid={`svg${type}`} color={getColor(type)}>
+          {getSymbol(type)}
+        </Circle>
+        <Value>{value}</Value>
+      </CircleValueWrapper>
+
+      <Title>{type}</Title>
+    </Wrapper>
+  )
+
+  function getColor(type) {
+    const color = {
+      Grow: function () {
+        return 'var(--main-yellow)'
+      },
+      Share: function () {
+        return 'var(--main-green)'
+      },
+      Spend: function () {
+        return 'var(--main-blue)'
+      },
+      default: function () {
+        return ''
+      },
+    }
+    return (color[type] || color['default'])()
+  }
+
+  function getSymbol(type) {
+    const symbol = {
+      Grow: function () {
+        return <GrowSymbol />
+      },
+      Share: function () {
+        return <ShareSymbol />
+      },
+      Spend: function () {
+        return <SpendSymbol />
+      },
+      default: function () {
+        return ''
+      },
+    }
+    return (symbol[type] || symbol['default'])()
+  }
+}
+const Wrapper = styled.div`
```

Comment:
How about a section instead of a div? And I don't like the name `FlexColumn`. If we'd use a section, we should name it after what it is, and not what it does? How about `DisplayBalanceWrapper`? 🙈

Reply:
If it's just for styling a div is okay, I think.

Reply:
I changed the name to `DisplayBalanceWrapper` and stuck with `div`

Reply:
I disagree here. It is not just for styling, this div is the container/parent of the DisplayBalance component and is sectioning the stuff (don't know how I could phrase it better) and therefore should be a sectioning element.

Reply:
I see your point @macfrei. As a section it provides semantical grouping for the component, which is what we want. 

Could this be generalized to always use a section as the outmost tag for components that are not just for layout (e.g. a PageLayout component that sets up a grid or something)?

Reply:
Yes, that's a really good explanation.

Reply:
It's settled. I changed the `div` to `section`

Reply:
I love these little conversation.